docs/shared/node-tutorial/1-code-generation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Node Nx Tutorial - Part 1: Code Generation'
+description: In this tutorial you'll create a backend-focused workspace with Nx.
+---
+
+{% callout type="check" title="Integrated Repo" %}
+This tutorial sets up an [integrated](/concepts/integrated-vs-package-based) repo. If you prefer a [package-based repo](/concepts/integrated-vs-package-based), check out the [Core Tutorial](/getting-started/core-tutorial).
+{% /callout %}
+
+# Node Nx Tutorial - Part 1: Code Generation
+
+In this tutorial you'll create a backend-focused workspace with Nx.
+
+## Contents
+
+- [1 - Code Generation](/node-tutorial/1-code-generation)
+- [2 - Nx Graph](/node-tutorial/2-nx-graph)
+- [3 - Task Running](/node-tutorial/3-task-running)
+- [4 - Workspace Optimization](/node-tutorial/4-workspace-optimization)
+- [5 - Summary](/node-tutorial/5-summary)
+
+## Your Objective
+
+For this tutorial, your objective is to create the initial architecture for a workspace with the following requirements:
+
+- it should contain an api built with express called: `products-api`.
+- it should contain a node cli application called: `products-cli`.
+- it should contain a library for interacting with a data-source called: `products-data-client`.
+- your `products-api` app should depend on the `products-data-client` library.
+- your `products-cli` app should also depend on the `products-data-client` library.
+
+![Our Workspace Requirements](/shared/node-tutorial/requirements-diagram.png)
+
+## Creating an Nx Workspace
+
+To generate your Nx Workspace, use the [`create-nx-workspace` script from npm](https://www.npmjs.com/package/create-nx-workspace):
+
+```bash
+npx create-nx-workspace@latest
+```
+
+When prompted, provide the following responses:
+
+```bash
+✔ Workspace name (e.g., org name)     · my-products
+✔ What to create in the new workspace · express
+✔ Application name                    · products-api
+```
+
+{% card title="Opting into Nx Cloud" description="You will also be prompted whether to add Nx Cloud to your workspace. We won't address this in this tutorial, but you can see the introduction to Nx Cloud for more details." url="/nx-cloud/intro/what-is-nx-cloud" /%}
+
+Once the script is complete, you can open up the `my-products` directory that was created, and see that your first project has been added to the workspace, located in `apps/products-api`.
+
+## Adding Another Application to Your Workspace
+
+Next you'll use [Nx generators](/plugin-features/use-code-generators) to generate the required `products-cli` application.
+
+The following syntax is used to run generators:
+
+![Nx Generator Syntax](/shared/node-tutorial/generator-syntax.png)
+
+To run the node application generator and create your `products-cli` application, run the command `npx nx g @nrwl/node:app products-cli`:
+
+```bash
```

Comment:
```bash is not correct I think. The output isn't bash

Reply:
I'll reach out in the docs channel tomorrow AM. We had mentioned trying no language specified on the code fences, but somehow not having a language makes it looks worse:

![Screen Shot 2022-10-13 at 12 44 21 AM](https://user-images.githubusercontent.com/3788405/195533981-c3f9e498-c253-4054-bbe8-cd46f7b8108d.png)

(`terminal` and `console` didn't work for me either)